CONTRIBUTING.md

```diff
@@ -3,15 +3,17 @@
 ## Running Fondue Locally
 
 1. Clone the Git repository
-2. Install the dependencies
+2. Install the dependencies with
```

Comment:
Maybe you could remove the `with` in that sentence or finish it. It's triggering my OCDs. 😅

Reply:
I forgot that `with` there. Sorry and thanks :D